layers/+intl/chinese/packages.el

```diff
@@ -26,7 +26,14 @@
 (defun chinese/init-fcitx ()
   (use-package fcitx
     :init
-    (fcitx-evil-turn-on)))
+    (fcitx-evil-turn-on)
+    :config
+    (setq fcitx-active-evil-states '(insert emacs hybrid))
+    (fcitx-default-setup)
+    (fcitx-prefix-keys-add "M-m" "C-M-m")
+    (if (eq system-type 'gnu/linux)
+        (setq fcitx-use-dbus t)))
```

Comment:
This option may lead to command lag on linux:

https://github.com/cute-jumper/fcitx.el/issues/30

Reply:
I see. I made it an option by adding a layer variable. README.org is also updated.